### ISSUE TITLE
[RFE] Platform UI - Functionality to provide schema names when listing tables in the UI to differentiate similar table names

### ISSUE THREAD

**Issue by adilakram87** [yugabyte/yugabyte-db#7163]
Issue :  Need to be able to differentiate similar table names within same database having different schema on UI. please see comments below.

Table listings in the UI have the table name and the keyspace (database) name. However in YSQL it is possible to have two tables with the same name but different schemas in the same database

So I get

Table Keyspace
customer. postgres
customer postgres

Where I really need to see the schema name to differentiate

Table Keyspace
schemaA.customer. postgres
schemaB.customer postgres

Customer is looking at 25 schemas in 1 database so this would become a large problem if they adopt this pattern

**Comment by adilakram87** [comment]
@datamgmt Tagging you so you know

**Comment by mjoshi0923** [comment]
@Arnav15 current get tables call returns following fields.
`
{
  isIndexTable: false
  keySpace: "ybdemo_keyspace"
  sizeBytes: 1384963903
  tableName: "cassandrauniquesecondaryindex"
  tableType: "YQL_TABLE_TYPE"
  tableUUID: "722446a0-4ed0-4b7f-8c81-647941343a24"
}
`

Can you help me to understand which field belongs to schema name? or I am missing something?

**Comment by mjoshi0923** [comment]
@adil246 Can you provide a screen shot of that you're expecting and what is current view?

**Comment by kkg-yb** [comment]
@adil246  could you provide the above requested screenshot?

**Comment by adil246** [comment]
@kkg-yb please see the actual description of the issue. 
This is a feature request. Today, UI doesn't display table schema for YSQL which supports same table name in different schema in same database. For a large application which can have many schemas, it can become difficult to differentiate tables with same name in different schemas. In the attached screenshot, both the tables are in different schema in same database.

```
ybtest=# select table_name, table_schema from information_schema.tables where table_schema not in ('pg_catalog', 'information_schema');
 table_name | table_schema 
------------+--------------
 tab01      | sch01
 tab01      | sch02
(2 rows)
```

![tables_in_same_schema](https://user-images.githubusercontent.com/78443978/118910782-b1d02b80-b8ea-11eb-9610-50cd83878cd4.png)

**Comment by kkg-yb** [comment]
@adil246 so what you are asking for is to add a schema name column to the display, right?

**Comment by adil246** [comment]
@kkg-yb Yes, this is beneficial if someone migrating from vanilla postgres. We can also list as schema name.ojbect_name.
